site2/docs/functions-runtime.md

```diff
@@ -270,7 +270,7 @@ For example, if you use token authentication, you need to configure the followin
 ```Yaml
 clientAuthenticationPlugin: org.apache.pulsar.client.impl.auth.AuthenticationToken
 clientAuthenticationParameters: file:///etc/pulsar/token/admin-token.txt
-configurationStoreServers: zookeeper-cluster:2181 # auth requires a connection to zookeeper
```

Comment:
There is no need to change.

Reply:
Once #13782 will be merged to master, you can ignore this review.

Reply:
> Once #13782 will be merged to master, you can ignore this review.

Agree. Thanks for pointing this out.